BookingsApi/BookingsApi.Domain/VhoNonAvailability.cs

```diff
@@ -18,5 +18,6 @@ public VhoNonAvailability(long id)
         {
             Id = id;
         }
+        public bool? Deleted { get; set; }
```

Comment:
as per previous comment, think it would make more sense for this to be non-nullable, true or false.

Reply:
done